src/engines/kotor/types.h

```diff
@@ -152,6 +152,19 @@ enum SubRace {
 	kSubRaceWookie
 };
 
+enum EquipmentSlot {
```

Comment:
You're adding `EquipmentSlot` here in the last commit, but you're already using it in an earlier commit, 8b567034ae121bc5d78e514c854f6b4be4d6944b. Which means that the tree at 8b567034ae121bc5d78e514c854f6b4be4d6944b won't compile.

I'd really like every single commit to be compilable. Especially for "git bisect", when looking for which commit introduced a regression, non-compilable commits are a nuisance.

Please reorder your commits and/or move changes to previous commits and/or split a few things into their own commits (and move them up the stack), so that each commit in the PR compiles.